src/microbe_stage/MembraneShapeGenerator.cs

```diff
@@ -479,6 +484,30 @@ private void GenerateMembranePoints(Vector2[] hexPositions, int hexCount, Membra
             startingBuffer[i] = closestOrganelle + movement;
         }
 
+        // Multicellular matrix
```

Comment:
Does this approach actually work or (in terms of making a good visual end result)? This is not how I would try to approach the intercellular matrix graphics at all (I'd make a separate mesh or a set of meshes to connect the cells), so I don't really want to spend a bunch of time trying to help debug this approach if it doesn't end up working in the end.

Reply:
I can't really tell you if it looks good as I can't get it to work in the first place, that's why I need help, so it's up to you. I don't really like the idea of some filler meshes inbetween cells as it isn't quite clear what such thing is (cell? connective tissue?) but if you really want that, I'm ready to scrap this solution

Reply:
I'm not ready to make a decisions on this, but my guess is that stretching the individual membranes will be a very hard to thing to get right. Though we don't know for certain until both approaches are tried as to what they can achieve. Sadly though I don't think it is good use of my limited time right now to try to get the code in this PR working, so that's up to you for now. Or if some other programmer is willing to help, that could be another way this PR gets working.